lms/djangoapps/ora_staff_grader/serializers.py

```diff
@@ -157,10 +159,55 @@ class Meta:
             "gradeStatus",
             "lockStatus",
             "score",
+            "email",
+            "fullname",
         ]
         read_only_fields = fields
 
 
+class AssessmentScoresMetadataSerializer(serializers.Serializer):
+    """
+    Assessment Score metadata for displaying assessment table in ESG
+    """
+    type = serializers.CharField()
+    score = serializers.IntegerField()
+    score_type = serializers.CharField()
+
+    class Meta:
+        fields = [
+            "type",
+            "score",
+            "score_type",
+        ]
+        read_only_fields = fields
+
+
+class AssessmentMetadataSerializer(serializers.Serializer):
+    """
+    Assessment metadata for displaying assessment table in ESG
+    """
+    idAssessment = serializers.CharField()
+    grader_name = serializers.CharField(allow_null=True)
+    grader_username = serializers.CharField(allow_null=True)
+    grader_email = serializers.CharField(allow_null=True)
+    assesmentDate = serializers.DateTimeField()
```

Comment:
Can you rename this variable to 
```suggestion
    assesment_date = serializers.DateTimeField()
```